src/base/static/js/views/app-view.js

```diff
@@ -946,16 +948,10 @@ export default Backbone.View.extend({
     );
 
     store.dispatch(setCurrentTemplate("map"));
-
-    if (mapPosition) {
-      emitter.emit(constants.MAP_TRANSITION_EASE_TO_POINT, {
-        coordinates: mapPosition.coordinates,
-        zoom: mapPosition.zoom,
-      });
-    }
+    store.dispatch(setAddPlaceButtonVisibility(true));
```

Comment:
@goldpbear Is there a reason why this line was re-added? I initially removed this line here: https://github.com/mapseed/platform/pull/1190/files

I have another PR that removes this line again: https://github.com/mapseed/platform/pull/1198/files
You can see the regression that it causes on pbdurham staging.